lib/engine/game/g_1835/entities.rb

```diff
@@ -0,0 +1,420 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G1835
+      module Entities
+        COMPANIES = [
+            {
+              name: 'Nürnberg-Fürth',
+              sym: 'NF',
+              value: 100,
+              revenue: 5,
+              desc: 'Nürnberg-Fürth Bahn, Director of AG may lay token on L14 north or south, 10% Bayern',
+              abilities: [{ type: 'shares', shares: 'BY_2' }, { type: 'no_buy' }],
+              color: nil,
```

Comment:
Set color, otherwise the UI will go a bit wonky.

Reply:
removed